docs/simpleinjector/gettingStarted.md

```diff
@@ -0,0 +1,17 @@
+# Getting Started with `Functional.CQS.AOP.IoC.SimpleInjector.Installation`
+
+Download the `Functional.CQS.AOP.IoC.SimpleInjector.Installation` package from NuGet, installing it in the project acting as your application's [composition root](http://blog.ploeh.dk/2011/07/28/CompositionRoot/).  Then, in your application's startup code (which holds a reference to the `SimpleInjector` container):
+
+```
+// register all CQS handlers contained in the specified assemblies
+// all CQS handler implementations will be registered with the specified lifestyle
+container.RegisterAllFunctionalHandlers(Lifestyle.Singleton, typeof(CompositionRootClass).Assembly, typeof(InfrastructureComponentClass).Assembly, ...)
```

Comment:
Instead of `CompostionRootClass` and `InfrastructureComponentClass`, would it make sense to say `ClassInFirstAssemblyContainingFunctionalCqsHandlers` and `ClassInSecondAssemblyContainingFunctionalCqsHandlers`?

That might make it clearer to a reader that they want to plug in all the assemblies containing their handlers, not the assembly containing the composition root or some other assembly.

Reply:
Most of the documentation refers to defining `Functional.CQS.AOP` components within the composition root, so I think that is fine.  The comment above the snippet should make it clear that the registrar picks up all CQS handlers in the assemblies supplied.

Reply:
Ah, sorry, yes, the comments are clear. 

I just looked at the specific classes named in the code and started thinking:
> Wait, why am I naming the assembly of my CompositionRoot? Is that a required parameter before I list the assemblies containing CQS handlers? What is an `InfrastructureComponentClass` and where do I find it? Is `InfrastructureComponentClass` defined elsewhere (eg. in this documentation, or in the source code for this project)?

Basically, the two class names distracted me from the comments and from the continued documentation after this snippet.

Reply:
`CompositionRootClass` will typically be `Startup` / `Program`, as composition will occur in the same assembly as other bootstrapping.  That assembly will not contain any handlers (usually), but will define some infrastructural concerns related to them (caching, metrics, etc).

`InfrastructureComponentClass` would be from one of the assemblies defining CQS handler implementations.